central/scannerdefinitions/handler/handler.go

```diff
@@ -38,24 +39,31 @@ import (
 )
 
 const (
-	definitionsBaseDir = "scannerdefinitions"
+	// rootDir is the root directory in which all Scanner data is written.
+	rootDir = "scannerdefinitions"
 
-	// scannerDefsSubZipName represents the offline zip bundle for CVEs for Scanner.
-	scannerDefsSubZipName  = "scanner-defs.zip"
-	scannerUpdateURLSuffix = "diff.zip"
+	// scannerDefsFile is the name of the file which contains Scanner v2 data.
+	scannerDefsFile = "diff.zip"
```

Comment:
Since we are using `V4` for Scanner V4, do you think it would make things easier to read if we also use `V2` for things related to Scanner V2?
```suggestion
	// scannerV2DefsFile is the name of the file which contains Scanner v2 data.
	scannerV2DefsFile = "diff.zip"
```